mojo/core/node_channel.cc

```diff
@@ -873,6 +873,11 @@ void NodeChannel::OfferChannelUpgrade() {
 #endif
 }
 
+void NodeChannel::Release() {
+  recordreplay::Assert("[RUN-1050] NodeChannel::Release");
```

Comment:
@sgeisenh sorry for a little bit of red tape, but Brian asked us to follow [these guidelines](https://github.com/replayio/backend/blob/master/docs/runtime/crashes-and-divergences.md#crash-diagnosis-getting-started), specifically (3). And Jason specifically cares about point (4) (and it's generally useful for triage to follow these anyway).

Reply:
Done.